src/components/Addbook.js

```diff
@@ -0,0 +1,24 @@
+import { useDispatch } from 'react-redux';
+
+function Addbook() {
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <button
+        type="button"
+        onClick={() => dispatch({
+          type: 'Add_Book',
+          payload: {
+            id: '',
+            title: '',
+            author: '',
+          },
+        })}
+      >
+        Submit
+      </button>
+    </div>
```

Comment:
- [ ] Add book functionality is not working, Please make sure to pass values entered by the user in dispatch and on submit button, the added book should display on the screen.